.github/workflows/release.yml

```diff
@@ -614,6 +646,42 @@ jobs:
           python test.py
         working-directory: test
 
+  validate_python_macos_arm:
+    name: Test python on MacOS M1
+    needs: [build_macos_arm_wheels]
+    runs-on: [self-hosted, m1]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set version env
+        id: version
+        run: echo "::set-output name=oso_version::$(cat VERSION)"
+      - name: Download oso python wheels from package run
+        uses: actions/download-artifact@v1
+        with:
+          name: wheel
+      # 3.8 is the earliest python for M1
+      - name: "test python 3.8"
+        run: |
+          arch -arm64 /opt/homebrew/opt/python@3.8/bin/python3 -m venv venv38
+          arch -arm64 ./venv38/bin/pip install oso==${{ steps.version.outputs.oso_version }} -f ../wheel --no-deps --no-index
+          arch -arm64 ./venv38/bin/pip install oso==${{ steps.version.outputs.oso_version }} -f ../wheel
+          arch -arm64 ./venv38/bin/python test.py
+        working-directory: test
+      - name: "test python 3.9"
+        run: |
+          arch -arm64 /opt/homebrew/opt/python@3.9/bin/python3 -m venv venv39
+          arch -arm64 ./venv39/bin/pip install oso==${{ steps.version.outputs.oso_version }} -f ../wheel --no-deps --no-index
+          arch -arm64 ./venv39/bin/pip install oso==${{ steps.version.outputs.oso_version }} -f ../wheel
+          arch -arm64 ./venv39/bin/python test.py
+        working-directory: test
+      - name: "test python 3.10"
```

Comment:
If we validate 3.10 here should we do it on the other ones as well?

Reply:
yeah, we need to build 3.10 for the other ones too, gonna do that seperately